scripts/build/Dockerfile.linux32.tmpl

```diff
@@ -3,29 +3,22 @@ ARG CC=gcc
 COPY scripts/ci/apt-install /bin/apt-install
 
 RUN apt-install \
-	libnet-dev \
-	libnl-route-3-dev \
 	$CC \
 	bsdmainutils \
 	build-essential \
 	git-core \
 	iptables \
-	libaio-dev \
-	libcap-dev \
 	libgnutls28-dev \
 	libgnutls30 \
-	libnl-3-dev \
-	libprotobuf-c-dev \
-	libprotobuf-dev \
 	libselinux-dev \
 	pkg-config \
 	protobuf-c-compiler \
 	protobuf-compiler \
-	uuid-dev \
 	python3-minimal
 
 COPY . /criu
 WORKDIR /criu
+RUN apt-install $(sed 's/\#.*$//' contrib/apt-packages.txt)
```

Comment:
Same comment here. It would be good to include all required dependencies in the TXT file. The following page provides more information: https://criu.org/Installation